src/mesg.c

```diff
@@ -155,6 +171,18 @@ send_paste(char *paste)
 	UNUSED(paste);
 }
 
+static int send_cmd_exists(char *cmd) {
+        int index = 0;
+
+        /* 46 - Number of commands in the irc_commands array. */
+        for ( index = 0; index < 46; index++ ) {
```

Comment:
46 doesn't really need to be hardcoded here, just use something like sizeof(irc_commands) / sizeof(*irc_commands)